apps/aesophia/src/aeso_ast_infer_types.erl

```diff
@@ -341,7 +352,8 @@ infer_letrec(Env, {letrec, Attrs, Defs}) ->
             Expect = typesig_to_fun_t(TypeSig),
             unify(Got, Expect, {check_typesig, Name, Got, Expect}),
             solve_field_constraints(),
-            %% io:format("Checked ~s : ~s\n", [Name, pp(dereference_deep(Got))]),
+	    ?PRINT_TYPES("Checked ~s : ~s\n",
```

Comment:
Is this a tab character?!

Reply:
Yes, I'll get rid of it next time I look at the module.